src/GeneralStateTestsFiller/stChainId/chainIdGasCostFiller.json

```diff
@@ -0,0 +1,55 @@
+{
+    "chainIdGasCost": {
+      "env": {
+        "currentCoinbase": "2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+        "currentDifficulty": "0x20000",
+        "currentGasLimit": "10000000000",
+        "currentNumber": "1",
+        "currentTimestamp": "1000",
+        "previousHash": "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+      },
+      "expect": [
+        {
+          "indexes": {
+            "data": -1,
+            "gas": -1,
+            "value": -1
+          },
+          "network": [">=Istanbul"],
+          "result": {
+            "1000000000000000000000000000000000000000": {
+              "storage": {
+                "0x01": "3"
+              }
+            }
+          }
+        }
+      ],
+      "pre": {
+        "1000000000000000000000000000000000000000": {
+          "balance": "",
+          "//code": "record the gas, run the operation, then record the gas again.  Drop output, subtract 2 for GAS, what is left is the cost of the operation, store in storage 0x01.",
+          "//codeLLL": "(asm GAS CHAINID GAS SWAP1 POP SWAP1 SUB 5 SWAP1 SUB 0x01 SSTORE)",
```

Comment:
substract 2  or 5 ? 
looks a good asm code template for gasCost checkup for each opcode